Marlin/Marlin_main.cpp

```diff
@@ -2160,6 +2160,14 @@ static void clean_up_after_endstop_or_probe_move() {
     #if ENABLED(BLTOUCH)
       if (set_bltouch_deployed(true)) return true;
     #endif
+	
+    #if ENABLED(PROBING_HEATERS_OFF)
+      // Wait for bed to heat back up between probing
+      while(thermalManager.degBed() < thermalManager.degTargetBed()) 
```

Comment:
Could be replaced with isHeatingBed()

logic of that method:

`FORCE_INLINE static bool isHeatingBed() { return target_temperature_bed > current_temperature_bed; }`

Which essentially does the same thing. Will update.